mix/ClientModel.cpp

```diff
@@ -350,6 +350,10 @@ void ClientModel::showDebuggerForTransaction(ExecutionResult const& _t)
 			CompiledContract const* contract = contracts[s.codeIndex];
 			AssemblyItem const& instruction = codeItems[s.codeIndex][instructionIndex];
 
+			std::stringstream str;
+			str << instruction.getLocation().sourceName;
+			qDebug() <<  QString::fromStdString(str.str());
```

Comment:
I will delete that later, I have to fix one other bug before.

2015-03-26 11:06 GMT+01:00 Arkadiy Paronyan <notifications@github.com>:

> In mix/ClientModel.cpp
> <https://github.com/ethereum/cpp-ethereum/pull/1413#discussion_r27200183>:
>
> > @@ -350,6 +350,10 @@ void ClientModel::showDebuggerForTransaction(ExecutionResult const& _t)
> >  			CompiledContract const* contract = contracts[s.codeIndex];
> >  			AssemblyItem const& instruction = codeItems[s.codeIndex][instructionIndex];
> >
> > +			std::stringstream str;
> > +			str << instruction.getLocation().sourceName;
> > +			qDebug() <<  QString::fromStdString(str.str());
>
> Do we really need that in debug output?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/cpp-ethereum/pull/1413/files#r27200183>.
>